pkg/deploy/controller/configchange/controller.go

```diff
@@ -1,162 +0,0 @@
-package configchange
```

Comment:
Can you add a commit in the middle that moves the controller code, then have a second commit that makes changes?  That makes this much easier to review and will also help trace ownership of changes.